android-sdk/src/test/java/com/checkout/android_sdk/PaymentFormTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.checkout.android_sdk
+
+import androidx.test.core.app.ApplicationProvider
+import com.checkout.android_sdk.Utils.Environment
+import com.checkout.eventlogger.CheckoutEventLogger
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.RobolectricTestRunner
+
+
+@RunWith(RobolectricTestRunner::class)
+class PaymentFormTest {
+
+    private lateinit var mockFramesLogger: FramesLogger
+
+    private lateinit var sdkLoggerMock: CheckoutEventLogger
+
+    private lateinit var paymentForm: PaymentForm
+
+
+    @Before
+    internal fun setUp() {
+        paymentForm = PaymentForm(ApplicationProvider.getApplicationContext())
+        sdkLoggerMock = mock(CheckoutEventLogger::class.java)
+        mockFramesLogger = mock(FramesLogger::class.java)
+        mockFramesLogger.initialise(ApplicationProvider.getApplicationContext(),
+            Environment.SANDBOX,
+            sdkLoggerMock)
```

Comment:
do we need to call initialise here? FramesLogger has been mocked

Reply:
No, we don't require it. I will remove it.

Reply:
No, we don't require it. I will remove it.